.buildkite/core.rayci.yml

```diff
@@ -10,6 +10,17 @@ steps:
     depends_on: corebuild
     job_env: corebuild
 
+  - label: ":ray: core: redis tests"
+    instance_type: large
+    parallelism: 6
+    commands:
+      - bazel run //ci/ray_ci:test_in_docker -- //python/ray/tests/... //python/ray/dag/... python/ray/autoscaler/v2/... core 
+        --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}" --parallelism-per-worker 3
+        --test-env=TEST_EXTERNAL_REDIS=1
```

Comment:
sometimes I wonder if they actually need to run all the tests against all these different variations. shouldn't they have an interface or something?

Reply:
i'm talking to data team first to see if we can simplify a lot of test environments (they are testing against a lot of pyarrow versions, etc.); hopefully can find something there